tvb_build/notebook_runner.py

```diff
@@ -65,7 +65,7 @@ def execute_notebook(notebook):
         'launching_bids_adapter.ipynb',
         'model_generation_using_dsl.ipynb',
         'RateML_CUDA_on_HPC.ipynb',
-        'RateML_Python_TVB.ipynb',
+        'exploring_K_ION_Exchange_model.ipynb',
```

Comment:
`exploring_K_ION_Exchange_model.ipynb` can be put back in the run. Then we can merge